hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/request/key/TestOMKeyCreateRequest.java

```diff
@@ -138,81 +140,94 @@ public void preExecuteRejectsInvalidReplication() {
   public void testValidateAndUpdateCache() throws Exception {
     when(ozoneManager.getOzoneLockProvider()).thenReturn(
         new OzoneLockProvider(keyPathLockEnabled, enableFileSystemPaths));
-    OMRequest modifiedOmRequest =
-        doPreExecute(createKeyRequest(false, 0));
 
-    OMKeyCreateRequest omKeyCreateRequest =
-            getOMKeyCreateRequest(modifiedOmRequest);
+    String[] validKeyNames = {
+        keyName,
+        OM_SNAPSHOT_INDICATOR + "abc/" + keyName,
+        "a/" + OM_SNAPSHOT_INDICATOR + "/b/c/" + keyName
+    };
+    for (String validKeyName : validKeyNames) {
```

Comment:
The only changes I added in this tests is just to create a String array of validKeyNames, then iterate them through as keyName in original test. (also add indent to original code block due to the new for-loop)